doc/source/developer_reference/output_processing_api.rst

```diff
@@ -3,14 +3,61 @@
 Output Processing API
 =====================
 
-WA3 now has an output API that can be used to post process a run's
-:ref:`Output Directory Structure<output_directory>` which can be performed by using WA's
-``RunOutput`` object.
-
-Example:
-
-If we have an existing WA output called ``wa_output`` in the current working
-directory we can initialize a ``RunOutput`` as follows:
+WA3 now has an output API that can be used to post process a runs
+:ref:`Output Directory Structure <output_directory>` which can be performed by using WA's
+:class:`RunOutput` object.
+
+This object provides a simple way to access the various aspects of the run. A
+few of the most useful attributes of the ``RunOutput`` object are as follows:
+
+- :confval:`artifacts` - A list containing any run level artifacts (for example
+  logs, output generated by output processors etc.).
+- :confval:`augmentations` - A list of the augmentations that were enabled
+  during the run.
+- :confval:`basepath` - The name of the base WA output folder.
```

Comment:
s/folder/directory/

here and elsewhere  -- we should keep the terminology consistent.